test/JDBC/expected/fts-contains-vu-verify.out

```diff
@@ -616,6 +594,1294 @@ GO
+int#!#text
+1#!#one two three
+2#!#two three one
+3#!#one
+6#!#one`two
+7#!#one~two
+8#!#one!two
+9#!#one@two
+10#!#one#two
+11#!#one$two
+12#!#one%two
+13#!#one^two
+14#!#one&two
+15#!#one*two
+16#!#one-two
+17#!#one_two
+18#!#one+two
+19#!#one=two
+20#!#one\two
+21#!#one|two
+22#!#one;two
+23#!#one:two
+24#!#one'two
+25#!#one<two
+26#!#one>two
+27#!#one.two
+28#!#one?two
+29#!#one/two
+101#!#one :) two
+~~END~~
+
+
+select * from test_special_char_t where contains(name, '":one"');
+go
+~~START~~
+int#!#text
+1#!#one two three
+2#!#two three one
+3#!#one
+6#!#one`two
+7#!#one~two
+8#!#one!two
+9#!#one@two
+10#!#one#two
+11#!#one$two
+12#!#one%two
+13#!#one^two
+14#!#one&two
+15#!#one*two
+16#!#one-two
+17#!#one_two
+18#!#one+two
+19#!#one=two
+20#!#one\two
+21#!#one|two
+22#!#one;two
+23#!#one:two
+24#!#one'two
+25#!#one<two
+26#!#one>two
+27#!#one.two
+28#!#one?two
+29#!#one/two
+101#!#one :) two
+~~END~~
+
+
+select * from test_special_char_t where contains(name, '" :one"');
+go
+~~START~~
+int#!#text
+1#!#one two three
+2#!#two three one
+3#!#one
+6#!#one`two
+7#!#one~two
+8#!#one!two
+9#!#one@two
+10#!#one#two
+11#!#one$two
+12#!#one%two
+13#!#one^two
+14#!#one&two
+15#!#one*two
+16#!#one-two
+17#!#one_two
+18#!#one+two
+19#!#one=two
+20#!#one\two
+21#!#one|two
+22#!#one;two
+23#!#one:two
+24#!#one'two
+25#!#one<two
+26#!#one>two
+27#!#one.two
+28#!#one?two
+29#!#one/two
+101#!#one :) two
+~~END~~
+
+
+select * from test_special_char_t where contains(name, '"one:"');
+go
+~~START~~
+int#!#text
+1#!#one two three
+2#!#two three one
+3#!#one
+6#!#one`two
+7#!#one~two
+8#!#one!two
+9#!#one@two
+10#!#one#two
+11#!#one$two
+12#!#one%two
+13#!#one^two
+14#!#one&two
+15#!#one*two
+16#!#one-two
+17#!#one_two
+18#!#one+two
+19#!#one=two
+20#!#one\two
+21#!#one|two
+22#!#one;two
+23#!#one:two
+24#!#one'two
+25#!#one<two
+26#!#one>two
+27#!#one.two
+28#!#one?two
+29#!#one/two
+101#!#one :) two
+~~END~~
+
+
+select * from test_special_char_t where contains(name, '"one: "');
+go
+~~START~~
+int#!#text
+1#!#one two three
+2#!#two three one
+3#!#one
+6#!#one`two
+7#!#one~two
+8#!#one!two
+9#!#one@two
+10#!#one#two
+11#!#one$two
+12#!#one%two
+13#!#one^two
+14#!#one&two
+15#!#one*two
+16#!#one-two
+17#!#one_two
+18#!#one+two
+19#!#one=two
+20#!#one\two
+21#!#one|two
+22#!#one;two
+23#!#one:two
+24#!#one'two
+25#!#one<two
+26#!#one>two
+27#!#one.two
+28#!#one?two
+29#!#one/two
+101#!#one :) two
+~~END~~
+
+
+select * from test_special_char_t where contains(name, 'one@#$two');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Concurrent special characters in the full-text search condition are not currently supported in Babelfish)~~
+
+
+select * from test_special_char_t where contains(name, '"one $%#! two"');
+go
+~~ERROR (Code: 33557097)~~
+
+~~ERROR (Message: Concurrent special characters in the full-text search condition are not currently supported in Babelfish)~~
+
+
+select * from test_special_char_t where contains(name, '"one # $ % ^ two"');
```

Comment:
what if we remove "", do we still get the error?

Reply:
Without "", only single word is allowed i.e. no spaces in between are allowed, so if we add space it'll automatically throw `syntax error`